### PR TITLE
Merge intervals

### PR DESCRIPTION
**Pull request by JuliaGerasymenko**

### CODE REVIEW

**Comment by JuliaGerasymenko** [comment]
[https://leetcode.com/problems/merge-intervals/](https://leetcode.com/problems/merge-intervals/)
I don't how to optimize more than it's now.

File: mergeIntervals.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const merge = intervals => {
+  let len = intervals.length;
+  if (len === 1) return intervals;
```
Comment (DzyubSpirit) [inline mergeIntervals.js]:
```suggestion
  if (len <= 1) return intervals;
```

**Review by DzyubSpirit — COMMENTED** [review]
Then let's explore other solutions. What do you think the complexity of the current algorithm?

**Comment by JuliaGerasymenko** [comment]
Time complexity  = O(n^2)
Space complexity = O(n)

**Pull request closed without merging**